src/app/modules/InvitationManagement/InvitationManagement.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { PageSection, Title, PageSectionVariants } from '@patternfly/react-core';
+import { InvitationProfile } from '@app/modules/InvitationManagement/components';
+
+const InvitationManagement: React.FC = () => (
+  <PageSection variant={PageSectionVariants.light}>
+    {/* <Title headingLevel="h1" size="lg">
+      Resource Management
+    </Title> */}
+    <div style={{ marginLeft: '20%', marginRight: '20%' }}>
```

Comment:
Add .css file for any custom css. Would be better to use patternfly global css variable if available. Avoid to use inline css.
https://www.patternfly.org/v4/developer-resources/global-css-variables
We can explore patterfly layouts form form alignment etc. https://www.patternfly.org/v4/layouts/stack